src/applications/personalization/dashboard-2/components/claims-and-appeals/hooks/useHighlightedClaimOrAppeal.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import moment from 'moment';
+
+const isWithinPast30Days = date => {
```

Comment:
We should move this into a `dashboard` util file since I have also needed to determine the most recent prescription that [has a status of Submitted OR Filled within the last 30 days](https://github.com/department-of-veterans-affairs/vets-website/pull/16064/files#diff-e65c6e859bf0065fe8d70d20bb661f34857bd600b33c064289f6c1ef01aff296R15). Feel free to consider it out of scope for this PR though and we can tackle that later during any cleanups.

Reply:
Issue created. Feel free to add to it https://github.com/department-of-veterans-affairs/va.gov-team/issues/21395